numba_cuda/numba/cuda/tests/cudapy/test_cuda_array_interface.py

```diff
@@ -5,14 +5,14 @@
 
 from numba import vectorize, guvectorize
 from numba import cuda
-from numba.cuda.testing import unittest, ContextResettingTestCase, ForeignArray
+from numba.cuda.testing import unittest, CUDATestCase, ForeignArray
 from numba.cuda.testing import skip_on_cudasim, skip_if_external_memmgr
 from numba.cuda.tests.support import linux_only, override_config
 from unittest.mock import call, patch
 
 
 @skip_on_cudasim("CUDA Array Interface is not supported in the simulator")
-class TestCudaArrayInterface(ContextResettingTestCase):
+class TestCudaArrayInterface(CUDATestCase):
```

Comment:
I don't think context resets are needed here - some tests check the list of pending deallocations and flush it, but that should not require a context reset.